accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-blink-rules.ts

```diff
@@ -41,6 +43,8 @@ let a11yRulesBlink: Rule[] = [
         run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
             const ruleContext = context["dom"].node as Element;
             let textValue = RPTUtil.getInnerText(ruleContext);
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
Shouldn't be skipped - this is a visual issue.

Reply:
removed